addons/ingress-nginx-metrics/v1.6.0.yaml

```diff
@@ -0,0 +1,326 @@
+      # This has to be adapted if you change either parameter
+      # when launching the nginx-ingress-controller.
+      - "ingress-controller-leader-nginx"
+    verbs:
+      - get
+      - update
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - create
+  - apiGroups:
+      - ""
+    resources:
+      - endpoints
+    verbs:
+      - get
+      - create
+      - update
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    k8s-addon: ingress-nginx.addons.k8s.io
+  name: nginx-ingress-controller
+  namespace: kube-ingress
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: nginx-ingress-controller
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: system:serviceaccount:kube-ingress:nginx-ingress-controller
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  labels:
+    k8s-addon: ingress-nginx.addons.k8s.io
+  name: nginx-ingress-controller
+  namespace: kube-ingress
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: nginx-ingress-controller
+subjects:
+  - kind: ServiceAccount
+    name: nginx-ingress-controller
+    namespace: kube-ingress
+
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: nginx-default-backend
+  namespace: kube-ingress
+  labels:
+    k8s-app: default-http-backend
+    k8s-addon: ingress-nginx.addons.k8s.io
+spec:
+  ports:
+  - port: 80
+    targetPort: http
+  selector:
+    app: nginx-default-backend
+
+---
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: nginx-default-backend
+  namespace: kube-ingress
+  labels:
+    k8s-app: default-http-backend
+    k8s-addon: ingress-nginx.addons.k8s.io
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-default-backend
+  revisionHistoryLimit: 10
+  template:
+    metadata:
+      labels:
+        k8s-app: default-http-backend
+        k8s-addon: ingress-nginx.addons.k8s.io
+        app: nginx-default-backend
+    spec:
+      terminationGracePeriodSeconds: 60
+      containers:
+      - name: default-http-backend
+        image: k8s.gcr.io/defaultbackend:1.3
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+        resources:
+          limits:
+            cpu: 10m
+            memory: 20Mi
+          requests:
+            cpu: 10m
+            memory: 20Mi
+        ports:
+        - name: http
+          containerPort: 8080
+          protocol: TCP
+
+---
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ingress-nginx
+  namespace: kube-ingress
+  labels:
+    k8s-addon: ingress-nginx.addons.k8s.io
+data:
+  # use-proxy-protocol: "true"
+  use-proxy-protocol: "false"
+  client-body-buffer-size: "16k"
+  proxy-connect-timeout: 60
+  proxy-body-size: "10m"
```

Comment:
Can you also change the other versions (at least the `v1.6.0-gce`), just in case?

Reply:
Yes. Changed in ingress-nginx-metrics.